src/OrbitQt/orbitmainwindow.h

```diff
@@ -168,6 +169,8 @@ class OrbitMainWindow : public QMainWindow {
   [[nodiscard]] bool ConfirmExit();
   void Exit(int return_code);
 
+  void StadiaConnectionError(std::error_code error);
```

Comment:
It's a bit unclear if that function is an event handler which is reacting to a connection error or if this is actually an event source which is called from something internal. From the implementation I know it's the former. So maybe something like `OnStadiaConnectionError`?

Reply:
done